documentation/modules/adding-source-provider.adoc

```diff
@@ -8,19 +8,21 @@ ifdef::vmware[]
 
 You can add a VMware vSphere source provider by using the {ocp} web console.
 
+In {project-short} versions 2.6 and later, you can migrate VMs from vCenter, from an ESX/ESXi server that is not managed by vCenter, or from an ESX/ESXi server that is managed by vCenter but does not go through vCenter.
```

Comment:
@RichardHoch that's better but about the last part "but does not go through vCenter" - the idea is that MTV doesn't interact with vCenter in this situation (of a server that is managed by vCenter but the specified endpoint is that of the server). this phrasing makes it sound as the server doesn't go through vCenter